### ISSUE TITLE
Short url without index.php

### ISSUE THREAD

**Issue by iterprise** [benhutchins/docker-mediawiki#19]
Is it possible to use short url without index.php? If it isn't can you add this possibility?

**Comment by benhutchins** [comment]
Yes, you just need to configure the apache and mediawiki settings. I don't have an example for doing this, but there are guides from MediaWiki, and essentially you need to either build your own `Dockerfile` that ends one of my images to replace the apache configuration or use a volume to mount your apache configuration file.
